sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -642,6 +642,13 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val CBO_ENABLED =
```

Comment:
Is this meant for enabling the whole cbo framework or just for controlling how the plan statistics calculated?